### PR TITLE
feat(*):showBetween prop to item between two breakpoints 

### PR DESCRIPTION
**Pull request by jasongi**

### CODE REVIEW

File: ts/react-responsive-components.tsx [hunk 1]
```diff
@@ -119,7 +119,11 @@ export class ResponsiveRoot extends React.Component<ResponsiveRootProps, void> {
 export interface ResponsiveProps extends MixedInHoCProps {
 	currentBreakpoint?: Breakpoint
 	showAtOrAbove?: string,
-	showAtOrBelow?: string
+	showAtOrBelow?: string,
```
Comment (damons-sportsbet) [inline ts/react-responsive-components.tsx]: I am wondering if we can split the props into two mutually exclusive types, one with showAtOrAbove and showAtOrBelow and another type with showBetween

File: js/react-responsive-components.d.ts [hunk 2]
```diff
@@ -64,12 +64,13 @@ export interface ResponsiveChildProps {
  * to children below the top-level.
  *
  * Whichever flavour you opt for, you can conditionally hide or show anything inside it by
- * passing showAtOrAbove or showAtOrBelow as props to <Responsive>.
+ * passing showAtOrAbove, showAtOrBelow or showBetween as props to <Responsive>.
```
Comment (damons-sportsbet) [inline js/react-responsive-components.d.ts]: gotta fix this comment now

File: js/react-responsive-components.d.ts [hunk 3]
```diff
@@ -64,12 +64,13 @@ export interface ResponsiveChildProps {
...
  *
  * You need to pass in your breakpoints object to every <Responsive>, but you can use responsiveHoC
  * to do this for you.
  */
 export declare class Responsive extends React.Component<ResponsiveProps, void> {
     getComparisonBreakpoint(comparisonBreakpointName: string): Breakpoint;
+    isBreakpointWithinBounds(): boolean;
```
Comment (damons-sportsbet) [inline js/react-responsive-components.d.ts]: this doesn't need to be exposed externally does it?

File: js/react-responsive-components.js [hunk 4]
```diff
@@ -123,7 +123,7 @@ exports.ResponsiveRoot = ResponsiveRoot;
  * to children below the top-level.
  *
  * Whichever flavour you opt for, you can conditionally hide or show anything inside it by
- * passing showAtOrAbove or showAtOrBelow as props to <Responsive>.
+ * passing showAtOrAbove, showAtOrBelow or showBetween as props to <Responsive>.
```
Comment (damons-sportsbet) [inline js/react-responsive-components.js]: gotta fix this comment too

File: js/react-responsive-components.d.ts (hunk 2 above)
Comment (jasongi) [reply js/react-responsive-components.d.ts]: done

File: js/react-responsive-components.d.ts (hunk 3 above)
Comment (jasongi) [reply js/react-responsive-components.d.ts]: have made it private

File: js/react-responsive-components.js (hunk 4 above)
Comment (jasongi) [reply js/react-responsive-components.js]: done

**Review by damons-sportsbet — APPROVED** [review]

**Pull request merged**